### DISCUSSION TITLE
Change email client

### DISCUSSION THREAD

**Discussion by Marcel972** [freeplane/freeplane#794] [Help]
Hello, I use the mailto link and I want to  run Thunderbird by default instead of Outlook, how to do it? >Thanks

**Comment by rouilj** [comment]
I expect that to be handled by setting your default email program at the OS level. What OS are you running on?

**Comment by Marcel972** [comment]
My OS is W11, I checked the default program and you are right, it was not Thunderbird. Thank you!
